client/ayon_zbrush/api/plugin.py

```diff
@@ -40,6 +41,19 @@ def _collect_instances(self):
 
 class ZbrushCreator(Creator, ZbrushCreatorBase):
     def create(self, product_name, instance_data, pre_create_data):
+        instance_kwargs = {
+            "product_type": self.product_type,
+            "product_name": product_name,
+            "data": instance_data,
+            "creator": self
+        }
+        if hasattr(self, "product_base_type"):
+            signature = inspect.signature(CreatedInstance)
+            if "product_base_type" in signature.parameters:
+                instance_kwargs["product_base_type"] = (
+                    self.product_base_type
+                )
+        new_instance = CreatedInstance(**instance_kwargs)
```

Comment:
```suggestion
```

Reply:
From the creator. We should be carefull about this. Most of the logic related to product base types is not enforced. When host integration starts to support "custom" product type then all of this has to be requirement, but now we have to slowly transition to new api and changes without affecting existing productions.